tests/models/test_onnx.py

```diff
@@ -152,3 +142,52 @@ def to_numpy(tensor):
 
     # compare ONNX Runtime and PyTorch results
     assert np.allclose(to_numpy(torch_out), ort_outs[0], rtol=1e-03, atol=1e-05)
+
+
+def test_model_saves_with_sequence_input(tmpdir):
+    """Test that ONNX model saves when input is tuple of tensors"""
+    class CustomModel(BoringModel):
+        def forward(self, x, y=None):
+            return super().forward(x)
+
+    def _assert_onnx_export(model, input_sample):
```

Comment:
Could you add model_name there, just to make sure it actually creates a file and doesn't use the previous one. I guess `model.to_onnx` should throw an error if something is wrong, but just to make it cleaner.

Reply:
done.